### ISSUE TITLE
New Firebase

### ISSUE THREAD

**Issue by miker1423** [ziyasal/FireSharp#89]
Hello, I have an API which was built a year ago, the guys of google told me that I needed to update to the new version of the management dashboard. After the update I can't do any kind of action, I already set the new URL and the new Secret but it is not working. can someone help me?
thanks!

**Comment by aupp** [comment]
Are you sure you got the right database secret?
From the console:
Overview (the cog icon in the top right corner of the menu) -> Project Settings -> Service Accounts -> Database Secrets. Its not the same as the "Web api key"

**Comment by miker1423** [comment]
Yes, I'm using the correct secret

**Comment by robomotic** [comment]
Just to make it clear did you upgrade from the old console version to the new console version? I actually have a setup like that but didn't try it yet, will let you know.

**Comment by miker1423** [comment]
Yes, I upgraded the console version which also updates the API to access the DB.
